### ISSUE TITLE
2019-11-20 tc015 M3SD comp engine error when parameterset set to "PARAMETERDISPLAYLIST"

### ISSUE THREAD

**Issue by jhhughes256** [tensfeldt/openNCA#43]
### Issue
When using tc015 and using the value of `"PARAMETERDISPLAYLIST"` for `parameterset` the computation engine crashes. This does not cause an error for tc010, tc011, tc013 or tc014. Errors for tc012 and tc016 are documented separately due to their unique error messages.

### Error

```
parameterset <- "PARAMETERDISPLAYLIST"
```
```
Error in value[[3L]](cond) : 
  Error in `[<-.data.frame`(`*tmp*`, i, , value = c(12659123, 1e+05, 32700, : replacement has 56 items, need 57
For SDEID 12659123 
7. stop(paste0(e, "For SDEID ", unique(data_data[, map_data$SDEID])[i])) at run_M3_SD_computation.R#1659
6. value[[3L]](cond) 
5. tryCatchOne(expr, names, parentenv, handlers[[1L]]) 
4. tryCatchList(expr, classes, parentenv, handlers) 
3. tryCatch({
    tmp_df <- data_data[data_data[, map_data$SDEID] == unique(data_data[, 
        map_data$SDEID])[i], ]
    tmp_df <- tmp_df[order(tmp_df[, map_data$TIME]), ] ... at run_M3_SD_computation.R#814
2. run_M3_SD_computation(data = merged_data, map = map_data, method = method, 
    model_regex = model_regex, parameter_list = parameter_list, 
    return_list = return_list) at run_computation.R#2241
1. run_computation(data = d, map = mct, flag = flags, parameterset = parameterset) 
```

**Comment by rudraya** [comment]
This issue was addressed in 02b2470, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
### Update

This issue has been fixed as of 02b2470.
